src/SFA.DAS.ApprenticeFeedback.Jobs/Startup.cs

```diff
@@ -24,7 +25,6 @@ namespace SFA.DAS.ApprenticeFeedback.Jobs
     public class Startup : FunctionsStartup
     {
         internal const string EndpointName = "SFA.DAS.ApprenticeFeedback";
-
```

Comment:
Don't take away whitespace that has a use

Reply:
fixed